views/unsubscribe.hbs

```diff
@@ -1,48 +1,49 @@
 {{!< default }}
 <div class="sub-page-content">
-  <section class="unsubscribe-content section-wrapper">
-    {{#if unsubscribed }}
+  {{#if unsubscribed }}
+    <section class="unsubscribe-content section-wrapper">
       <h2 class="section-headline whole">
       You are no longer subscribed.
       </h2>
       <p class="whole sub-head">
       Your email is unsubscribed from Firefox Monitor. Thank you for using this service. Will you take a moment to answer one question about your experience?
       </p>
-  </section>
-  <section class="unsubscribe-survey section-wrapper drop-shadow">
-    <span class="secondary-title whole">Why are you unsubscribing from Firefox Monitor alerts?</span>
-    <div class="whole">
-      <form action="" method="post" id="unsubscribe-survey-form" class="form-group">
-        {{#each UNSUB_REASONS}}
-        <div class="radio-button-group">
-          <input id="reason{{ id_number }}" type="radio" name="unsubscribe-reason" value="{{ value }}" />
-          <span class="radio-dot"></span>
-          <label for="reason{{ id_number }}">{{ value }}</label>
-        </div>
-        {{/each}}
-        <div class="input-group-button">
-          <input id="unsubscribe" type="submit" class="button submit transparent-button" value="Unsubscribe" tabindex="0"/>
-          <img class="loader" src="/img/loader.gif" alt="loading data" />
-        </div>
-      </form>
-    </div>
-  </section>
-</div>
-    {{ else }}
-    <h2 class="section-headline whole">Unsubscribe from Firefox Monitor</h2>
-    <p class="whole sub-head">
-      This will remove your email from the Firefox Monitor list and you will no longer receive alerts when new breaches are announced.
-    </p>
-    <div class="half">
-      <form action="/user/unsubscribe" class="form-group" method="post" id="unsubscribe-form">
-        <input type="hidden" name="token" value="{{ token }}">
-        <input type="hidden" name="emailHash" value="{{ hash }}">
-        <div class="input-group-button">
-          <input type="submit" class="button submit transparent-button" value="Unsubscribe">
-          <img class="loader" src="/img/loader.gif" alt="loading data" />
-        </div>
-      </form>
-    </div>
+    </section><!--.unsubscribe-content.section-wrapper-->
+    <section class="unsubscribe-survey section-wrapper drop-shadow">
+      <span class="secondary-title whole">Why are you unsubscribing from Firefox Monitor alerts?</span>
+      <div class="whole">
+        <form action="" method="post" id="unsubscribe-survey-form" class="form-group">
+          {{#each UNSUB_REASONS}}
+            <div class="radio-button-group">
+              <input id="reason{{ id_number }}" type="radio" name="unsubscribe-reason" value="{{ value }}" tabindex="1" />
+              <span class="radio-dot"></span>
```

Comment:
For some reason I couldn't choose the reason by clicking on the radio dots themselves? I had to click on the text.

Reply:
Next on list to fix after emails are all presentable for going out in public.